x/blob/client/cli/payforblob.go

```diff
@@ -57,10 +69,24 @@ func CmdPayForBlob() *cobra.Command {
 	}
 
 	flags.AddTxFlagsToCmd(cmd)
+	cmd.PersistentFlags().String(FlagShareVersion, "0", "Specify the share version (default is 0)")
```

Comment:
if we define this flag as something other than a string, like an int for instance, will we be able to avoid having to do a conversion? Is there a different reason we need to use a string here?

Reply:
Thanks, changed to uint8.